src/enums.js

```diff
@@ -29,6 +29,7 @@ export const cropType = enumify([
   'TOMATO',
   'WATERMELON',
   'WHEAT',
+  'WEEDS',
```

Comment:
What do you think about switching from plural "weeds" semantics to singular ("weed") for both the `cropType` here and the `itemType` below in order to remain consistent with the other `cropType` and `itemType` semantics?